Tabloid/client/src/components/CategoryEditForm.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { FormGroup, Label, Button, Form, Input } from "reactstrap";
+import { editCategory, getCategory } from "../modules/categoryManager";
+import { useHistory, Link, useParams } from "react-router-dom";
+
+export const CategoryEditForm = () => {
+  const { categoryId } = useParams();
+  const [category, setCategory] = useState({
+    id: 1,
```

Comment:
Since 1 is possibly an id of an actual category in the db, this should be set to 0 as the initial state. 0 is the integer version of 'nothing' or 'empty string'

Reply:
Done